### ISSUE TITLE
Notification of when Remote Start/Lock could be available

### ISSUE THREAD

**Issue by Spiffieman** [home-assistant/home-assistant.github.io#18539]
### Feedback

Hi there. Whoever wrote this is awesome. I just got into HASS stuff and finding that my car has an integration is fantastic. How would I be notified when the integration is able to do the remote start/lock, etc? 

### URL

https://www.home-assistant.io/integrations/subaru/

### Version

2021.7.3

### Additional information

Thank you! Amazing work!

**Comment by G-Two** [comment]
Hi @Spiffieman, thanks for the feedback! I just put in a [PR](https://github.com/home-assistant/core/pull/52852) to add remote lock/unlock to the integration. Once it is approved, it will be merged into the main Home Assistant Core repo. There's a pretty good chance it'll be approved in time to make release 2021.8. Following that, I'll add services, including remote engine start/stop.

If you don't want to wait for the incremental updates, you can also use [HACS](https://hacs.xyz/) to add my [custom component](https://github.com/G-Two/homeassistant-subaru) integration which includes all the remote features already. My goal is to incrementally merge all of its features into Home Assistant Core.

**Comment by Spiffieman** [comment]
Hi Garrett -thank you! Do you know off hand why I keep on getting an " Message
malformed: expected dict for dictionary value @
data['sequence'][0]['data']" error message when trying to create a script
to call the remote start service? The full 17 character vin is in there.

sequence:
  - service: subaru.remote_start
    data: 4S4WMAPD*********
mode: single
alias: Start Subaru


The plan would be to use this to create a remote start button on a
dashboard.

- Andrew

**Comment by G-Two** [comment]
Looks like a YAML formatting issue in your code. Data is expected to be a dict, and vin is required key.
```
service: subaru.remote_start
data:
  vin: JF2ABCDE6L0000001
```
In Developer Tools you can play with the services to verify the integration works. There's a good example of a dashboard [here](https://github.com/G-Two/homeassistant-subaru/discussions/13) with all the remote services exposed as buttons.

**Comment by Spiffieman** [comment]
Awesome, thanks!

**Comment by frenck** [comment]
Thank you for reaching out.  We use GitHub for tracking issues, not for providing support.

If you are in need of support, you should try our [Community Forum](https://community.home-assistant.io) or join our [Discord chat server](https://discord.gg/c5DvZ4e).

You are more likely to get the answer you are looking for in those places.

Thanks! 👍
